assets/babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react",
+        "@babel/preset-typescript"
+    ],
+    "plugins": [
+        "@babel/plugin-proposal-class-properties"
+    ]
+}
```

Comment:
This file had to be renamed to support the new version of `babel-jest`. I'm not sure why.